src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -935,6 +935,8 @@ internal void ResolveConflict([NotNull]string docID, [CanBeNull]IConflictResolve
                         writeSuccess = SaveResolvedDocument(resolvedDoc, localDoc, remoteDoc);
                     });
                     
+                } catch (Exception ex) {
+                    throw ex;
```

Comment:
This doesn't change the behavior.  If there is no `catch` then the exception is thrown by default.